examples/foood/src/Editor.js

```diff
@@ -634,6 +674,32 @@ const RecipeEditor = ({
                 {/* ) : null} */}
             </div>
             {onDelete != null ? <DeleteButton onConfirm={onDelete} /> : null}
+
+            {text.ops[0].insert.length > 1 && getIngredientNames(text).length === 0 ? (
```

Comment:
it might be nice to extract this into a function, something like `shouldShowIngredientsAlert(text)`